app/models/experiment.rb

```diff
@@ -98,7 +98,7 @@ def completed?
 
   def get_statistics
     all = simulation_runs.count
-    sent = simulation_runs.where(to_sent: false, is_done: false).count
+    sent = simulation_runs.where(is_done: false, to_sent: false).count
```

Comment:
(Nie)stety trzymaja - "Hashes enumerate their values in the order that the corresponding keys were inserted."

2.2.1 :003 > {a: 1, b: 2}.to_json
 => "{\"a\":1,\"b\":2}"
2.2.1 :004 > {b:2, a: 1 }.to_json
 => "{\"b\":2,\"a\":1}"
2.2.1 :005 > {a: 1, b: 2}.keys
 => [:a, :b]
2.2.1 :006 > {b:2, a: 1 }.keys
 => [:b, :a]

Ten hash (lub dokument jsonowy) jest przekazywany do mongodb który wspiera coś o nazwie compound indexes - https://docs.mongodb.org/v3.0/core/index-compound/. Kolejnosc wystepowania kluczy jest istotna przy ich uzywaniu. 

Ten przypadek jest argumentem za tym aby wyszukiwania opakowywac w metody ktorych nazwy beda mowily o co chodzi a implementacja bedzie w 1 miejscu.